dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionResource.java

```diff
@@ -55,6 +63,8 @@
 @Path("/organizations")
 @Api("/organizations")
 @Produces(MediaType.APPLICATION_JSON)
+@Tag(name = "organizations", description = "Operations on Dockstore organizations.")
+@SecuritySchemes({ @SecurityScheme(type = SecuritySchemeType.HTTP, name = "bearer", scheme = "bearer", bearerFormat = "JWT") })
```

Comment:
Our tokens are not JWT. Is there another option for bearer format?

Reply:
I based the JWT thing off of the existing operation in the Swagger annotations. Looks like it is arbitrary. I'll just change it to BEARER as seen in the Constants.java file.

Reply:
Actually it is optional, I'll remove it.